supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/AwsBatchAsyncBackendJobExecutionActor.scala

```diff
@@ -460,9 +462,28 @@ class AwsBatchAsyncBackendJobExecutionActor(override val standardParams: Standar
       case unknown => throw new RuntimeException(s"Attempt to get terminal metadata from non terminal status: $unknown")
     }
   }
+  def hostAbsoluteFilePath(jobPaths: JobPaths, pathString: String): Path = {
+
+    val pathBuilders:List[PathBuilder]  = List(DefaultPathBuilder)
+    val path = PathFactory.buildPath(pathString, pathBuilders)
+    path match {
+      case _: DefaultPath if !path.isAbsolute => jobPaths.callExecutionRoot.resolve(path).toAbsolutePath
+      case _: DefaultPath if jobPaths.isInExecution(path.pathAsString) => jobPaths.hostPathFromContainerPath(path.pathAsString)
+      case _: DefaultPath => jobPaths.hostPathFromContainerInputs(path.pathAsString)
+    }
+  }
 
   override def mapOutputWomFile(womFile: WomFile): WomFile = {
-    womFileToPath(generateAwsBatchOutputs(jobDescriptor))(womFile)
+    configuration.fileSystem match {
+      case  AWSBatchStorageSystems.s3 =>
+        womFileToPath(generateAwsBatchOutputs(jobDescriptor))(womFile)
+      case _ =>
+        val hostPath = hostAbsoluteFilePath(jobPaths, womFile.valueString)
```

Comment:
I think this block can be a little bit simplified like this:
```suggestion
        val hostPath = hostAbsoluteFilePath(jobPaths, womFile.valueString)
        if (!hostPath.exists) throw new FileNotFoundException(s"Could not process output, file not found: ${hostPath.pathAsString}")
        val wfile = womFile mapFile { _ => hostPath.pathAsString }
        womFileToPath(generateAwsBatchOutputs(jobDescriptor))(wfile)
```

Reply:
done